index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoko WEB UI</title>
+    <link rel="shortcut icon" href="/favicon.ico" />
```

Comment:
`/favicon.ico` doesn't exist, and neither does the Web UI live at the root either. Prefix the icon with `/webui` to make it accessible from any url location within the Web UI.
```suggestion
    <link rel="shortcut icon" href="/webui/favicon.ico" />
```

Reply:
the `/webui/ ` prefix is automatically added by Vite with the `base` option.

Reply:
I did double check, and I can confirm that the favicon does work as expect.